test/http/rfc7230.cpp

```diff
@@ -43,8 +43,11 @@ class rfc7230_test : public beast::unit_test::suite
         {
             s.push_back(';');
             s.append(str(p.first));
-            s.push_back('=');
-            s.append(str(p.second));
+            if(! p.second.empty())
```

Comment:
I have a hard time getting excited about a microoptimization in unit test code...